scikit-imiage version bumped to 0.18.3 to use the one in EasyBuild
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/deeplabcut-2.2.0.6.orig/setup.py b/deeplabcut-2.2.0.6/setup.py
index 3bff4f2..2e0926b 100644
--- a/deeplabcut-2.2.0.6.orig/setup.py
+++ b/deeplabcut-2.2.0.6/setup.py
@@ -33,7 +33,7 @@ setuptools.setup(
         "networkx",
         "numpy",
         "pandas>=1.0.1",
-        "scikit-image>=0.17,<=0.18.1",
+        "scikit-image>=0.17,<=0.18.3",
         "scikit-learn",
         "scipy>=1.4",
         "statsmodels>=0.11",
